### PR TITLE
fix: Make updating async

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

**Comment by kristofbolyai** [comment]
@magicus @Incompleteusern can I get a review? I want to merge this before pinging people to use gh builds

**Review by DevScyu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -87,4 +87,3 @@
 * Unidentified Item Icons ([#271](https://github.com/Wynntils/Artemis/issues/271)) ([3d73347](https://github.com/Wynntils/Artemis/commit/3d73347b00718981381f3670a32ae09d35b5528f))
 
 ## 0.0.0 (2021-12-09)
-
```
Comment (magicus) [inline CHANGELOG.md:90]: Is this a spurious change triggered by the new build publishing tooling?

Comment (kristofbolyai) [reply CHANGELOG.md:90]: yeah + spotless, might be a one off though

Comment (magicus) [reply CHANGELOG.md:90]: Ok, we'll need to keep an eye on this so it does not start ping-ponging.

**Pull request merged**